crates/aptos/src/move_tool/mod.rs

```diff
@@ -1141,7 +1141,7 @@ impl CliCommand<TransactionSummary> for RunFunction {
     }
 }
 
-/// Run a Move function
+/// Run a view function
```

Comment:
Probably should provide more helpful information after it, but this is fine for now.